crates/rolldown/src/stages/generate_stage/compute_cross_chunk_links.rs

```diff
@@ -9,14 +9,14 @@ use rolldown_common::{
   NamedImport, OutputFormat, SymbolRef,
 };
 use rolldown_rstr::{Rstr, ToRstr};
-use rolldown_utils::rayon::{IntoParallelIterator, ParallelBridge, ParallelIterator};
+use rolldown_utils::rayon::{ParallelBridge, ParallelIterator};
 use rustc_hash::{FxHashMap, FxHashSet};
 
 type ChunkMetaImports = IndexVec<ChunkId, FxHashSet<SymbolRef>>;
 type ChunkMetaImportsForExternalModules =
   IndexVec<ChunkId, FxHashMap<ExternalModuleId, Vec<NamedImport>>>;
 type ChunkMetaExports = IndexVec<ChunkId, FxHashSet<SymbolRef>>;
-type IndexCrossChunkImports = IndexVec<ChunkId, FxHashSet<ChunkId>>;
+type IndexCrossChunkImports = IndexVec<ChunkId, Vec<ChunkId>>;
```

Comment:
How `Vec<ChunkId>` avoid duplicate imports to the same chunk?

Reply:
Yeah. Here should be `IndexSet`.

Reply:
I don't think so. If the order relies on the order of traversing symbols, we might break the order due to adjust the generated runtime symbols by ourselves.

Depending the order calculated by `sorting with modules` is much reliable than this.

Reply:
Ok. Let me revert it.

Reply:
Saw you were saying revert it back in https://github.com/rolldown/rolldown/pull/1002#issuecomment-2082796466. Did you mean rollup ensure the order of `imports` is the execution order of chunk?

Reply:
Yeah. The rollup using js `Set/Map`, the Set and Map have the  insertion order, the insertion order is stabled.

Reply:
I missing to revert it.